src/pages/Autolending/AutolendingOptOutPage.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+	<www-page :gray-background="true">
+		<div class="title-area">
+			<div class="row column">
+				<h1>Auto-lending</h1>
```

Comment:
This title area is duplicate code. If this gets approved to stay on this opt-out page then we can move this page and the autolending settings page to share the title area with a child/parent route

Reply:
This minor amount of duplication is fine for now. After initial launch of this page we can consider some enhancements to using a parent view instead.